core/src/test/scala/com/salesforce/op/filters/FeatureDistributionTest.scala

```diff
@@ -238,4 +241,12 @@ class FeatureDistributionTest extends FlatSpec with PassengerSparkFixtureTest wi
     intercept[IllegalArgumentException](fd1.jsDivergence(fd1.copy(name = "boo"))) should have message
       "requirement failed: Name must match to compare or combine feature distributions: A != boo"
   }
+
+  it should "not serialize cardEstimate field" in {
+    val fd1 = FeatureDistribution("A", None, 10, 1, Array(1, 4, 0, 0, 6),
+      Array.empty, Some(Moments(1.0)), Some(TextStats(Map("foo" -> 1, "bar" ->2))),
+      FeatureDistributionType.Scoring)
+
+    FeatureDistribution.toJson(Seq(fd1)) shouldNot contain theSameElementsAs "cardEstimate"
```

Comment:
actually the test seems incorrect, it won't fail without the fix:
```scala
scala> "blah cardEstimate blah" shouldNot contain theSameElementsAs "cardEstimate"
res2: org.scalatest.Assertion = Succeeded
```

Reply:
include is captures it:
```scala
scala> "blah cardEstimate blah" shouldNot include ("cardEstimate")
org.scalatest.exceptions.TestFailedException: "blah cardEstimate blah" included substring "cardEstimate"
  at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:343)
  at org.scalatest.Matchers$ShouldMethodHelper$.shouldNotMatcher(Matchers.scala:6731)
  at org.scalatest.Matchers$AnyShouldWrapper.shouldNot(Matchers.scala:7151)
  ... 36 elided

scala> "blah cardEstimat blah" shouldNot include ("cardEstimate")
res4: org.scalatest.Assertion = Succeeded
```

Reply:
Good catch! I guess `contain` here works differently from `contains` to string

Reply:
http://www.scalatest.org/user_guide/using_matchers#workingWithAggregations
theSameElementAs was not the right choice because it can generate false positive for a permutation of the string order, and it does not have to be consecutive.